interfaces/builtin/home.go

```diff
@@ -82,6 +82,7 @@ audit deny @{HOME}/bin/{,**} wl,
 
 const homeConnectedPlugAppArmorWithAllRead = `
 # Allow non-owner read to non-hidden and non-snap files and directories
+capability dac_override,
```

Comment:
Won't this allow write too?

Reply:
@bboozzoo - it does not override the apparmor 'owner' check and there are no non-owner writes in home, so, 'no', but if we allowed non-owner writes, then 'yes'. That said, looking at the forum topic again, I'm instead going to change this to dac_read_search.

Reply:
Thanks for explaining this in detail!